website/content/docs/configuration/storage/cockroachdb.mdx

```diff
@@ -63,4 +63,4 @@ storage "cockroachdb" {
 ```
 
 [cockroachdb]: https://www.cockroachlabs.com/
-[pglib]: https://godoc.org/github.com/lib/pq#hdr-Connection_String_Parameters
+[pgxlib]: https://pkg.go.dev/github.com/jackc/pgx/v4#hdr-Establishing_a_Connection
```

Comment:
There might be a better reference for what params the connection string can take. I wasn't able to find something in the pgx docs outside of the pgconn library it uses. We might be able to reference https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING.

Reply:
:+1: on that

Reply:
Added in https://github.com/hashicorp/vault/pull/15901/commits/ea62e91912cf3d9121928530fce66545e260a11e 👍